CMakeLists.txt

```diff
@@ -247,12 +247,14 @@ if(EMSCRIPTEN)
 endif()
 
 if(WIN32 AND NOT VCPKG_BUILD)
+if(NOT MINGW)
```

Comment:
Why no `AND NOT MINGW`?

Reply:
If you cross-compile from Linux, you will always use some form of MinGW.

Reply:
Ok... I actually didn't know that WIN32 is true when cross compiling, but still, why not merge both into a single if statement?

Reply:
`WIN32` is true. `NOT VCPKG_BUILD` is true. `NOT MINGW` is false. The final result is false. I afraid that it will break cross-compiling. I don't know anything about cross-compiling. I only use MSYS2 on Windows. But @tobbi raised the issue. I tried to do my best not to break it.

Reply:
Tested with `if(WIN32 AND NOT VCPKG_BUILD AND NOT MINGW)` on MSYS2 CLANG64, the command is still being executed. I'm not an expert in this so I can't debug to know why.

Reply:
Oh well. It's not a big deal anyway